go/cmd/vtbackup/vtbackup.go

```diff
@@ -220,13 +246,13 @@ func takeBackup(ctx context.Context, topoServer *topo.Server, backupStorage back
 	}()
 
 	// Start up mysqld as if we are mysqlctld provisioning a fresh tablet.
-	mysqld, mycnf, err := mysqlctl.CreateMysqldAndMycnf(tabletAlias.Uid, *mysqlSocket, int32(*mysqlPort))
```

Comment:
Everything from here downward was added in the branch this one is based on https://github.com/vitessio/vitess/pull/11334